Makefile

```
@@ -0,0 +1,315 @@
+ci-ui-test: deps generate-project
+	$(MAKE) ui-test SIMULATOR_UDID=$(SIMULATOR_UDID)
+
+# Generate Xcode Project (Helper for CI, can be used locally too)
+.PHONY: generate-project
+generate-project: deps
+	@echo "⚙️ Generating Xcode project using xcodegen..."
+	mint run xcodegen generate
+	@echo "✅ Xcode project generated."
+
+# === Package Tests ===
+.PHONY: test-packages
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)